tests/validation/test_gfssf_validator.py

```diff
@@ -0,0 +1,54 @@
+import unittest
+
+import numpy as np
+import pandas as pd
+
+import ballet.validation.gfssf_validator as gfv
+
+
+class GFSSFValidadatorTest(unittest.TestCase):
+    def setUp(self):
+        pass
+
+    def test_discrete_entropy(self):
```

Comment:
split these into separate test cases (unit testing a lot cleaner than the end-to-end testing)

i.e. `test_discrete_entropy_same_val`